yaml/buildings.yaml

```diff
@@ -19,7 +19,9 @@ globals:
       building_levels: {col: "building:levels"}
       building_min_levels: {col: "building:min_levels"}
       building_material: {col: "building:material"}
+      castle_type: {col: castle_type}
       height: {col: height}
+      historic: {col: historic}
```

Comment:
What values are present here in historic? Should they be turned into specific `kind` values instead?

Reply:
Yes, sure, I can do it. But I am not sure how. It is a very long time when I was changing it, so I cannot remember how this could be done. Thank you.